packages/react/src/components/TextArea/TextArea.tsx

```diff
@@ -26,63 +24,58 @@ interface TextAreaProps extends Omit<TextFieldBaseProps, 'endIcon' | 'startIcon'
   onChange?: (value: string) => void;
 }
 
-const TextArea = React.forwardRef<TextAreaElement, TextAreaProps>((props, forwardedRef) => {
-  const {
-    className,
-    helperTextProps = {},
-    inputProps: inputPropsProp = {},
-    inputRef,
-    labelProps: labelPropsProp = {},
-    onChange,
-    ...other
-  } = props;
-
-  const areaRef = React.useRef<HTMLTextAreaElement>(null);
+export const TextArea = React.forwardRef<typeof TextFieldBase, TextAreaProps>(
+  (props, forwardedRef) => {
+    const {
+      className,
+      helperTextProps = {},
+      inputProps: inputPropsProp = {},
+      inputRef,
+      labelProps: labelPropsProp = {},
+      onChange,
+      ...other
+    } = props;
 
-  const [inputValue, setInputValue] = useControlledState(props.value, props.defaultValue, () => {
-    //noop
-  });
+    const areaRef = React.useRef<HTMLTextAreaElement>(null);
 
-  const handleHeightChange = React.useCallback(() => {
-    const input = areaRef.current as HTMLTextAreaElement;
-    const prevAlignment = input.style.alignSelf;
+    const [inputValue, setInputValue] = useControlledState(props.value, props.defaultValue, () => {
+      //noop
```

Comment:
did you mean to leave this comment?

Reply:
yep, react aria expects a default function and eslint complains about empty functions. this was a way around it